src/components/Stepper/Stepper.tsx

```diff
@@ -1,13 +1,17 @@
 import React, { useState, useEffect, useCallback } from "react";
 import classnames from "classnames";
 import "./Stepper.scss";
-import {
-  SvgCheckCircle,
-  SvgCircleProgress,
-  SvgCircleDisabled,
-} from "../../assets/svg-components";
+import { SvgDone, SvgPriorityHigh } from "../../assets/svg-components";
 
-type ItemsType = Array<{
+export enum StepState {
```

Comment:
Would recommend to move this separate file like StepState.enum.ts or Stepper.enums.ts.